lib/game.rb

```diff
@@ -2,51 +2,50 @@
 require_relative './constants.rb'
 
 class Game
-    attr_reader :board, :state, :players, :active_player_index
+  attr_reader :board, :state, :players, :active_player_index
 
-    def initialize(args={})
-        @state = GAME_STATE_NEW
-        @players = args.fetch(:players)
-        @active_player_index = args.fetch(:active_player, random_player_index)
-        @board = args.fetch(:board, Board.new)
-    end
+  def initialize(args = {})
+    @state = GAME_STATE_NEW
+    @players = args.fetch(:players)
+    @active_player_index = args.fetch(:active_player, random_player_index)
+    @board = args.fetch(:board, Board.new)
+  end
 
-    def reset
-        @state = GAME_STATE_NEW
-        @active_player_index = random_player_index
-        @board.reset
-    end
+  def reset
+    @state = GAME_STATE_NEW
+    @active_player_index = random_player_index
+    @board.reset
+  end
 
-    def set_state(state)
-        @state = state
-    end
+  def set_state(state)
+    @state = state
+  end
```

Comment:
Why not user `attr_reader` and `attr_writer`?

Reply:
Also, "set state" is not a great method... The name is not very specific and the context is way too large (What is the state? What does it encompass?)